access.lua

```diff
@@ -334,6 +334,7 @@ end
 
 if hlp.is_logged_in() then
     -- serve ynhpanel files
+    serveAsset("/ynh_userinfo.json", "js/ynh_userinfo.js")
```

Comment:
This doesn't work ... Accessing domain.tld/ynh_userinfo.json doesn't return the expected info... Also this function `serveAsset` will then apply public-caching ... but since this json is meant to contain private info, this should probably be cached as private instead